layers/meta-balena-raspberrypi/recipes-core/images/balena-image.bbappend

```diff
@@ -12,8 +12,10 @@ SDIMG_KERNELIMAGE:raspberrypi3-64 ?= "kernel8.img"
 SDIMG_KERNELIMAGE:raspberrypi0-2w-64 ?= "kernel8.img"
 
 # Increase Root File system size
-# ToDo (Waqar): Find correct size and fix issues
-IMAGE_ROOTFS_SIZE:revpi-connect-s ?= "2785280"
+IMAGE_ROOTFS_SIZE:revpi-connect-s ?= "319488"
```

Comment:
Changed the available space allocated from 278MB to 319 MB to have enough space with the variable IMAGE_ROOTFS_SIZE